relationships/synthesis/INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,33 @@
+relationships:
+  - name: k8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "CONTAINER" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            lookup: true
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+          identifier:
+            fragments:
+              - value: "k8s:"
+              - attribute: k8s.cluster.name
+              - value: ":"
+              - attribute: k8s.namespace.name
+              - value: ":pod:"
+              - attribute: k8s.pod.name
```

Comment:
nit: Just wondered if the k8s.pod.name is also part of a container's identifier ?